tests/committee_tests.py

```diff
@@ -156,13 +169,13 @@ def test_committee_filters(self):
 
         # checking one example from each field
         filter_fields = (
-            # ('committee_id', ['bartlet', 'ritchie']),
-            # ('state', ['CA', 'DC']),
+            ('committee_id', ['bartlet', 'ritchie']),
```

Comment:
Good point, will change now. It's just harder to talk about the West Wing without using names!